web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificate.test.tsx

```diff
@@ -129,6 +143,144 @@ describe("<AnyTimeActionTaxClearanceCertificateReviewElement />", () => {
       fireEvent.click(screen.getByText(Config.taxClearanceCertificateStep3.editButtonText));
       expect(screen.getAllByRole("tab")[1]).toHaveAttribute("aria-selected", "true");
     });
+
+    it("renders tab one as complete when on tab two", () => {
+      const business = generateBusinessWithEmptyTaxClearanceData();
+      renderComponent({ business });
+      const firstTab = screen.getAllByRole("tab")[0];
+      const secondTab = screen.getAllByRole("tab")[1];
+      expect(firstTab).toHaveAttribute("aria-label", expect.stringContaining("State: Incomplete"));
+
+      fireEvent.click(secondTab);
+      expect(secondTab).toHaveAttribute("aria-selected", "true");
+      expect(firstTab).toHaveAttribute("aria-label", expect.stringContaining("State: Complete"));
+    });
+
+    it("renders tab one as complete when on tab three", () => {
+      const business = generateBusinessWithEmptyTaxClearanceData();
+      renderComponent({ business });
+      const firstTab = screen.getAllByRole("tab")[0];
+      const thirdTab = screen.getAllByRole("tab")[2];
+      expect(firstTab).toHaveAttribute("aria-label", expect.stringContaining("State: Incomplete"));
+
+      fireEvent.click(thirdTab);
+      expect(thirdTab).toHaveAttribute("aria-selected", "true");
+      expect(firstTab).toHaveAttribute("aria-label", expect.stringContaining("State: Complete"));
+    });
+
+    it("renders tab one as incomplete when viewing tab one and not all required data is provided", () => {
```

Comment:
test should be reevaluated based on changes made

Reply:
Can you clarify what needs to be re-evaluated? I'm not I'm seeing any issues.